src/components/builder.scss

```diff
@@ -3,9 +3,19 @@
     flex-wrap: wrap;
     flex-direction: row;
     justify-content: space-between;
+    font-family: Borda;
     flex-basis: 75%;
 }
 
+.button{
+    font-family: Borda;
```

Comment:
Isn't this covered by the app.scss?

Reply:
It is. I have removed this in my most recent commit